File: docs/mexc/mexc-api-sdk-main/mexc-api-sdk-main/examples/python/cancel_orders.py
from mexc_client import MexcClient

client = MexcClient(api_key='YOUR_API_KEY', is_testnet=True)

result = client.cancel_orders({
    'ids': ['101716841474621953', '108885377779302912', '108886241042563584']
})

print(result)

File: docs/mexc/mexc-api-sdk-main/mexc-api-sdk-main/examples/nodejs/cancelOrders.js
import { MexcClient } from './MexcClient.js';

const client = new MexcClient({ apiKey: 'YOUR_API_KEY', isTestnet: true });

const result = await client.cancelOrders({
  ids: ["101716841474621953", "108885377779302912", "108886241042563584"]
});

console.log(result);